lua/refactoring/config/init.lua

```diff
@@ -71,6 +71,7 @@ function Config:new(...)
         prompt_func_param_type = default_prompt_func_param_type,
         printf_statements = default_printf_statements,
         print_var_statements = default_print_var_statements,
+        extract_new_var_statements = default_extract_new_var_statements,
```

Comment:
Should we perhaps call this `extract_var_statements`, just to be consistent with the other settings (each one references the name of the operation they're associated with)?

Reply:
Sure let me update.

Reply:
Updated for new name, I'll merge.